projects/plugins/jetpack/class.jetpack.php

```diff
@@ -3395,6 +3396,14 @@ public static function do_version_bump( $version, $old_version ) {
 	 */
 	public static function set_update_modal_display() {
 		self::state( 'display_update_modal', true );
+
+	}
+
+	/**
+	 * Enqueues the block library styles.
+	 */
+	public static function enqueue_block_style() {
+		wp_enqueue_style( 'wp-block-library' );
```

Comment:
Would it be possible to limit this to the different pages of the Jetpack dashboard, where the modal is displayed? Something like this (I have not tested it)?

```suggestion
	 * @param string $hook Page hook.
	 */
	public static function enqueue_block_style( $hook ) {
		if ( 0 === strpos( $hook, 'jetpack' ) ) {
			wp_enqueue_style( 'wp-block-library' );
		}
			
```

Reply:
This is a really good idea, and I've added a check for the Jetpack dashboard in commit 70668bf.

I also realized that I was only enqueuing the block stylesheet immediately after the version update happens. However, the Jetpack dashboard might not be visited immediately after the update. The user could navigate to other admin pages before visiting the JP dashboard. We use a cookie value to determine whether the update modal should be displayed, so I added a check for that cookie value in commit 70668bf. If it exists, we enqueue the stylesheet.